services/thermal_mgr/thermal_mgr.c

```diff
@@ -43,18 +43,36 @@ void initThermalSystemManager(lm75bd_config_t *config) {
 
 error_code_t thermalMgrSendEvent(thermal_mgr_event_t *event) {
   /* Send an event to the thermal manager queue */
-
+  xQueueSend(thermalMgrQueueHandle, event, 0);
```

Comment:
Add error handling to this
- [ ] check that queue is not null, returning invalid state
- [ ] check that event is not null
- [ ] check the return value against pdTRUE for success